slapp/transforms/transform_pipeline.py

```diff
@@ -204,6 +226,22 @@ def run(self, db_conn: query_utils.DbConnection):
             imageio.imsave(outline_path, outline, transparency=0)
             imageio.imsave(full_outline_path, full_outline, transparency=0)
 
+            # add the scale to the outlines they will toggle with the outline
+            # element on the web app
+            scale_position = (0, outline.shape[1])
+            scale_position_full = (0, full_outline.shape[1])
+            font_file_path = Path(self.args['font_file'])
+
+            outline_with_scale = add_scale(outline_path, scale_position,
```

Comment:
I think it would be better if your `add_scale` function accepts an Image or numpy array rather than a path. Because now the transform pipeline writes to disk, reads immediately and then overwrites. It is not a huge overhead, but, it's best to just do things in memory.

Reply:
Changed to in memory, I had made a previous comment to Kat that this was difficult but I resolved that. I had forgotten to convert to RGBA before drawing in PIL which causes errors. Everything is now in memory and only written once.